### PR TITLE
[MOB-1920] - Include javadoc in travis

### PR DESCRIPTION
**Pull request by Ayyanchira**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -69,6 +69,7 @@ script:
   - ./gradlew :iterableapi:jacocoTestDebugUnitTestReport :app:jacocoDebugTestReport :iterableapi:connectedCheck :app:jacocoDebugAndroidTestReport
 
 after_script:
+  - ./gradlew :iterableapi:javadoc
```
Comment (vbabenkoru) [inline .travis.yml:72]: Does this actually fail if you have invalid javadoc? Have you tested it?

Comment (Ayyanchira) [reply .travis.yml:72]: I saw it getting run in the logs and not failing as the docs were intact.. Will try to push some wrong documentation in this branch and see if its failing or not.

Comment (Ayyanchira) [reply .travis.yml:72]: Force pushed with wrong docs.. Lets see 🤞

**Comment by Ayyanchira** [comment]
Checked... it does not fail even with wrong docs 😦.. Trying to run that gradlew command separately in regular script section

**Pull request merged**